libursa/src/utils/macros.rs

```diff
@@ -17,6 +17,20 @@ macro_rules! array_copy {
     };
 }
 
+#[cfg(any(
+    feature = "bls_bls12381",
+    feature = "ed25519",
+    feature = "ed25519_asm",
+    feature = "ecdh_secp256k1",
+    feature = "ecdh_secp256k1_native",
+    feature = "ecdh_secp256k1_asm",
+    feature = "ecdsa_secp256k1",
+    feature = "ecdsa_secp256k1_native",
+    feature = "ecdsa_secp256k1_asm",
+    feature = "x25519",
+    feature = "x25519_asm",
+    feature = "wasm"
```

Comment:
You need to add the feature `cl` in this list.

Reply:
I don't think impl_bytearray is used in the cl feature. It compiles without it.